web_widget_image_paste_clipboard/static/src/js/web_widget_image_paste_clipboard.js

```diff
@@ -0,0 +1,39 @@
+odoo.define('web_widget_image_paste_clipboard.FieldBinaryImage', function(require) {
+    "use strict";
+
+    var basicFields = require('web.basic_fields');
+
+    basicFields.FieldBinaryImage.include({
+        _render: function() {
+            this._super.apply(this, arguments);
+            var self = this;
+
+            function handlePaste(event) {
```

Comment:
Can you move this to the class, so it can be overridden / extended by other modules.